packages/commcare/src/Adaptor.js

```diff
@@ -113,21 +98,20 @@ export function submitXls(formData, params) {
     data.append('create_new_cases', create_new_cases);
 
     console.log('Posting to url: '.concat(url));
-    return http
-      .post({
-        url,
-        data,
-        headers: {
-          ...data.getHeaders(),
-          Authorization: `ApiKey ${username}:${apiKey}`,
-        },
-      })(state)
-      .then(response => {
-        return { ...state, data: { body: response.data } };
-      })
-      .catch(err => {
-        throw { ...err, config: {}, request: {} };
-      });
+
+    const response = await request({
+      state,
+      method: 'POST',
+      path: path,
+      data,
+      header: {
+        ...data.getHeaders(),
+        Authorization: `ApiKey ${username}:${apiKey}`,
```

Comment:
Yes. I had a conversation with @mtuchi  but if I change it to use basic auth it might create disruptions in jobs using the adaptor so I think creating another issue to handle this would be ideal

Reply:
Great - raised as #554